tasks/gitHelpers.ts

```diff
@@ -0,0 +1,196 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { spawnSync } from 'child_process';
+import * as fs from 'fs';
+import * as path from 'path';
+import { Octokit } from '@octokit/rest';
+import { NugetPackageInfo } from './offlinePackagingTasks';
+import { PlatformInformation } from '../src/shared/platform';
+import { platformSpecificPackages } from './offlinePackagingTasks';
+
+export interface GitOptions {
+    userName?: string;
+    email?: string;
```

Comment:
these two options appear to never be used.  I doubt they should even be options - should likely just hardcode to whatever the localization pipeline already uses